koji/collection-mash-split.py

```diff
@@ -407,11 +408,13 @@ def main():
             "el8": "el8",
             "rhel7": "el7",
             "rhel6": "el6",
-            "rhel5": "el5",
             "sles11": "sles11",
             "sles12": "sles12",
         }
 
+        if LooseVersion(version) < LooseVersion("2.3"):
```

Comment:
In the past we've been just doing `if version in ['2.0', '2.1', '2.2']:`

But using `LooseVersion` is also valid.

Reply:
My specific thinking was, reading the code says "We are dropping el5 in 2.3", instead of "we are still shipping el5 in 2.0, 2.1, 2.2"

Reply:
Since I was the person who wrote those `if version in ...` parts I can tell you that it was easy to do it this way when I unified the mash-script-per-version into this single script. I don't have a particular preference, but we have a bunch of these comparisons. It'd be nice if the styles were the same.

Reply:
Personally, I think utilizing LooseVersion opens up some avenues for minimizing need to manage arrays of versions, so i'll propose doing one of these, and let me know which one you would like to move forward with:

a) switch all comparisions in this pr
b) leave this pr as is, open a new pr fixing the other comparisions
c) switch the above comparison to the "in array", and open a new pr switching all the comparisons.

Reply:
I'm leaning to option c.

Reply:
updated.